src/Mapper.cpp

```diff
@@ -2072,14 +2185,14 @@ namespace karto
     m_Initialized = false;
   }
 
-  kt_bool Mapper::Process(Object* /*pObject*/)
+  kt_bool Mapper::Process(Object*  /*pObject*/)  // TODO(lucbettaieb)  What even is this doing?
```

Comment:
The parameter is required to match the function signature -- but if it had a name, then we would get an unused variable warning. This syntax works around both issues. (We can remove the comment)